src/Tools/dotnet-trace/CommandLine/Commands/CollectLinuxCommand.cs

```diff
@@ -11,15 +11,16 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Diagnostics.NETCore.Client;
+using Microsoft.Diagnostics.Tools.Common;
 using Microsoft.Internal.Common.Utils;
 
 namespace Microsoft.Diagnostics.Tools.Trace
 {
-    internal static partial class CollectLinuxCommandHandler
+    internal partial class CollectLinuxCommandHandler
     {
         private static bool s_stopTracing;
         private static Stopwatch s_stopwatch = new();
-        private static LineRewriter s_rewriter = new() { LineToClear = Console.CursorTop - 1 };
+        private static LineRewriter s_rewriter;
         private static bool s_printingStatus;
```

Comment:
If testing caused multiple instances of CollectLinuxCommandHandler to run in parallel, would we want those instances to be sharing any of these static fields? I'm guessing we'd want all these fields to be instanced too.